CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Improvements:
 - Allow editing Kits when presets are in use. [#1965](https://github.com/microsoft/vscode-cmake-tools/issues/1965)
 - Launch the target in the default terminal. [PR #2311](https://github.com/microsoft/vscode-cmake-tools/pull/2311) [@michallukowski](https://github.com/michallukowski)
 - Allow launching targets in parallel. [#2240](https://github.com/microsoft/vscode-cmake-tools/issues/2120) [@ColinDuquesnoy](https://github.com/ColinDuquesnoy)
+- Add commands to get preset names. [#2433](https://github.com/microsoft/vscode-cmake-tools/pull/2433)
```

Comment:
As Microsoft FTE I left my name out, but if you want me to add it I can.

Reply:
Good question.  It is intentional that we only list non-MSFT folks in our changelog.

Reply:
please modify [#2433] to [PR #2433] and move it below 1.11 before merging.